demos-europe/demosplan-core
# feat (DPLAN-12571): Add new original statement list

### Ticket
[DPLAN-12571](https://demoseurope.youtrack.cloud/issue/DPLAN-12571/ADO-Issue-20425-EWM-Integration-Part-1.1.-Erganzung-der-Originalstellungnahmen-in-der-EWM-Integration)

This PR extracts the original statements list from the assessment table. The API 2.0 is used for this (OriginalStatementResourceType).

### How to review/test
code review, test the list in the interface and check the acceptance criteria.

To test the list in the interface, go to the statement list, then select 'original statements' in the top navigation.





### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
